source/integrate/jira-interoperability.rst

```diff
@@ -50,6 +50,8 @@ A Mattermost system admin must perform the following steps in Mattermost.
     - **Hide issue descriptions and comments**: Show or hide issue descriptions and comments from subscription and webhook messages.
     - **Enable slash command**: Enable or disable slash command autocompletion to guide users through available ``/jira`` slash commands.
     - **Display Subscription name in notifications**: Show or hide subscription name in notification messages posted to a channel.
+    - **Admin API Token**: Set an `API token  <https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/>`_ to get notified for comments and issue creation events, even when the user triggering the event isn't connected to Jira, and set up Autolink. API tokens must be created using an admin Jira account; otherwise, notifications won't be delivered for projects the user can't access, and Autolink won't work.
+    - **Admin Email**: Set Admin email to set up Autolink for the Jira plugin.
```

Comment:
This is not required just for the auto link but also for the event notifications, please update it

Reply:
@raghavaggarwal2308 Fixed